cypress/e2e/home.cy.ts

```diff
@@ -1,20 +1,26 @@
-describe('Movapp - Home page', () => {
+/// <reference types="Cypress" />
+
+describe('Test Movapp - Home page', () => {
   beforeEach(() => {
-    cy.visit('/');
+    cy.visit("https://www.movapp.cz/");
```

Comment:
Doménu přesuntě do environment variable at můžou testy běžet pak oproti testovacímu prostředí na vercelu nebo localhost (by default).

Na to je environment proměnná, kterou nastavíš `CYPRESS_BASE_URL`:
- CI v pull requestu kde je vercel to testuješ oproti vercelu
- Pak bude workflow oproti produkci (tam dáš toto co si nyní dal)

V Gitlabu to pak vypadá třeba takto:

```
script:
- TZ=Europe/Prague AUTO_LOGIN_USER=${AUTO_LOGIN_USER} AUTO_LOGIN_PASSWORD=${AUTO_LOGIN_PASSWORD} CYPRESS_BASE_URL=${APP_URL} ./node_modules/.bin/cypress run --headless --browser chrome --reporter mocha-junit-reporter
```

V `cypress.config.js` dej `e2e.baseUrl: 'http://localhost'`

Například:

```json

    e2e: {
        specPattern: 'cypress/e2e/**/*.cy.{js,jsx,ts,tsx}',
        baseUrl: 'http://localhost',
        setupNodeEvents: setupNode(width, height, tempValues),
        experimentalStudio: true
    },
```